ReactiveXComponentTest/IntegrationTests/RabbitMqCommunicationTest.cs

```diff
@@ -135,11 +139,60 @@ public void SendEvent_GivenAStateMachineAMessageAndAVisibility_ShouldSendMessage
             Check.That(contenu).IsEqualTo(_message);
         }
 
-        private void PublishMessage(Visibility visibility)
+        [TestCase(Visibility.Private)]
+        [TestCase(Visibility.Public)]
+        public void AddCallback_GivenAcomponentAStateMachineAndACallback_ShouldCreateSubscriptionAndReceiveMessageOnCallback_Test(Visibility visibility)
         {
-            string privateCommunicationIdentifier = visibility == Visibility.Private? "PrivateCommincationIdentifier" : string.Empty;
+            var subscirber = CreateSubscriber(visibility);
+            const string component = "Component";
+            const string stateMachine = "stateMachine";
+
+            subscirber.AddCallback(component, stateMachine, MessageReceivedUpdated);
+
+            PublishMessage(visibility);
+
+            string label = null;
+            const int timeoutReceive = 10000;
+            var lockEvent = new AutoResetEvent(false);
+            
+            MessageReceived += instance =>
```

Comment:
What's the purpose of MessageReceived ? 
Why not just check the result with MessageReceivedUpdated ?